RELEASE.md

```diff
@@ -240,6 +246,14 @@ Update the documentation to recommend using the new version:
 $ find . -name "*.md" -exec sed -i "s/v1.0.0/v1.1.1/g" {} +
 ```
 
+## Send a PR to reference Actions at main
+
+Send a PR to reference the internal Actions at `@main`. You can use:
+
+```shell
+find .github/ -name '*.yaml' -o -name '*.yml' | xargs sed -i 's/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@_YOUR_RELEASE_TAG_*/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@main/'
```

Comment:
Is this sed replacement not backwards? It looks like it replaces `uses: slsa-framework/slsa-github-generator/.github/actions/action_name@vX.Y.Z` with `uses: slsa-framework/slsa-github-generator/.github/actions/action_name@main` but shouldn't it be the other way around for release PRs?

Reply:
Ah, I see. I missed the change above. This one is to set it back to main.

Reply:
After the release, we revert to using `@main` for development until the next release. The reasoning is that during development, we want to use the most recent code to catch problems before the next release.